libcnb-test/src/build_config.rs

```diff
@@ -251,6 +252,8 @@ impl BuildConfig {
 pub enum BuildpackReference {
     /// References the buildpack in the Rust Crate currently being tested.
     Crate,
+    /// References a libcnb.rs buildpack within the Rust Workspace that needs to be packaged into a buildpack
+    LibCnbRs(BuildpackId),
     /// References another buildpack by id, local directory or tarball.
     Other(String),
```

Comment:
I find the current names a bit confusing - is there a way we can improve these?

Crate -> CurrentCrate?
LibCnbRs -> LocalCrate?


...or something else?

Reply:
i'm open to suggestions. if we want to keep the language close to Rust-isms (e.g.; `Crate`, `WorkspaceMember`) that kind of makes sense. it is a little weird when those combine with Buildpack-isms though because not everything is a crate. 

For example:
* using `WorkspaceMember("heroku/nodejs-engine")` or `LocalCrate("heroku/nodejs-engine")` for a component buildpack where `heroku/nodejs-engine` is the Buildpack Id & `buildpacks/nodejs-engine` is the Workspace path.
* using `WorkspaceMember("heroku/nodejs")` or `LocalCrate("heroku/nodejs")` for a composite buildpack where `heroku/nodejs` is the Buildpack Id & `meta-buildpacks/nodejs` is the Workspace path but it isn't a crate.

to avoid that mismatch what do you think of `WorkspaceBuildpack(BuildpackId)` @edmorley?

Reply:
Oh true, of course the meta-buildpacks aren't a crate.

`WorkspaceBuildpack` sounds good to me.

What do you think about also renaming `Crate` to `CurrentCrate`?